### ISSUE TITLE
NVDA becomes very sluggish in the Firefox Downloads window

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#853]
**Reported by elliott94 on 2010-08-23 18:53**
If there are several (roughly 15 or above) downloads in the Firefox Downloads list, NVDA becomes very sluggish, and the only way to improve performence is to remove all downloads from the list. Tested with several other screen readers, and I can't seem to produce the same results. This only happens on my netbook which i guess is understandable; it doesn't have as many resources as desktop PC's, but JAWS seems to manage fine, not that that says much.
This was tested on a netbook with 1 GB of RAM and a 1.33 GHZ processer, and Firefox 3.6.8 with XP SP3, however performence was the same with older versions as well.

**Comment by jcsteh** [comment]
No response from anyone else, and we (nor Mozilla) can reproduce this any more. Closing.

**Comment by derekriemer** [comment]
Unrelated, but #4417

I have a fix, is this still occurring?


On 6/23/2016 7:02 PM, James Teh wrote:
>
> No response from anyone else, and we (nor Mozilla) can reproduce this 
> any more. Closing.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/853#issuecomment-228229692>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivfJo07QxDw7KMfLHYqlE1UHwFq1wks5qOyyrgaJpZM4I9YJQ>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194
